portfolio/src/main/java/com/google/sps/servlets/StringServlet.java

```diff
@@ -21,7 +21,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
         quotations.add("Education is the passport to the future," 
-                          +" for tomorrow belongs to those who prepare for it today. -Malcolm X");
+                              + " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```

Comment:
I was actually referring to this :)
```java
        quotations.add("Education is the passport to the future," 
            + " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```
or this
```java
        quotations.add("Education is the passport to the future," +
            " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```

Reply:
Ah, haha, ok I'll fix that now